features/hwb.yml

```
@@ -0,0 +1,9 @@
+name: HWB
+description: The `hwb()` CSS function picks colors using hue, whiteness and blackness channels. It is used by many color pickers.
```

Kinda like not mentioning support information in descriptions, I think it's a good idea to avoid other non-durable information like popularity. I don't think we lose too much cutting it out like this:

```suggestion
description: The `hwb()` CSS function picks colors using hue, whiteness and blackness channels.
```

---

I was wanting to add the primary use case or reason it exists since this seems more narrowly focused, but I'm fine removing it for your reasons and also since the other color types don't have similar info.